design/design.md

```diff
@@ -0,0 +1,175 @@
+# Asteria dApp Design
+
+## Introduction
+This document describes the technical design of the Asteria dApp - the script UTxOs involved, the operations that take place both during the game and in the setup phase, and the necessary validators and minting policies.
+
+There will be a single script UTxO for the `Pot`, several `PelletState` UTxOs and a `ShipState` UTxO for every user. The `Pot` UTxO locks the ada amount paid by each user when creating a ship, and it's position on the board is always assumed to be (0,0). Both `PelletState` and `ShipState` UTxOs have their positions specified in the datum. In order to identify valid game UTxOs, the admin will deposit a special token in the `PelletState` and `Pot` UTxOs when creating them.
```

Comment:
We could add at the end of this paragraph that the "admin token" is also used for parameterizing the pot and fuel validators, so we could have different "versions" of the game, each one with a different admin token.